src/poc_noresp_ping.nim

```diff
@@ -54,10 +57,17 @@ proc setUpNodes(numNodes: int): seq[Switch] =
 
       # Extract private key and multiaddress
       let (multiAddrStr, _, _, _, libp2pPrivKey) = getMixNodeInfo(node)
-      let multiAddr = MultiAddress.init(multiAddrStr.split("/p2p/")[0]).value()
+
+      let multiAddr = MultiAddress.init(multiAddrStr.split("/p2p/")[0]).valueOr:
+        error "Failed to initialize MultiAddress", err = error
+        return
```

Comment:
Using the function from prev comment, it's likely we can do some quick changes to avoid having to initialize a multiaddress here:
1. `MixNodeInfo` `multiAddr` field should be a `Multiaddress` instead of a `string`. The multiaddress initialization can happen when you are loading the node info from a file.
2. `getMixNodeInfo` would then return a `Multiaddress` instead of a string.

That would allow you to do something like:
```
let multiAddr = mixNodeMultiAddr.stripPeerId()
```